lib/tasks/normalize_resources.rake

```diff
@@ -0,0 +1,12 @@
+namespace :data do
+  desc 'Update resources with nil quantity/used fields'
+  task normalize_resources: :environment do
+    Resource.where('used is null or quantity is null').each do |resource|
```

Comment:
Not able to find any other option
`Resource.where('used is null or quantity is null').each do |resource|` 
Reason: http://stackoverflow.com/questions/4340517/how-to-compare-null-in-rails-active-record-query